variables.tf

```diff
@@ -31,6 +37,16 @@ variable "databases" {
   default = {}
 }
 
+variable "additional_geo_locations" {
```

Comment:
Could this be named just `geo_locations` now? Seeing as we now only have a single `geo_location` block.

Reply:
> Since this is a non backwards compatible change
I reverted this - it should be backwards compatible, ref: https://github.com/avinor/terraform-azurerm-cosmosdb-mongodb/blob/f4f557d417623ad03a79464ebeebff1a1e58fbb2/main.tf#L79-L91.

Reply:
examples and tests pass.